src/components/KeyboardShortcutsModal/KeyboardShortcutsModal.js

```diff
@@ -0,0 +1,85 @@
+import { upperFirst } from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Button,
+  Modal,
+  ModalFooter,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import commands from '../../commands';
+import commandsGeneral from '../../commandsGeneral';
```

Comment:
Apologies for delay - This looks great!! The only thing I'll think we'll need to do here to get this 'centralized' is to use some of these module-specific imports as props passed into the 'KeyboardShortcutsModal component where it is instanciated in `src/index.js` . - probably have an 'allCommands' prop that would accept the value you currently assemble internally from concatenating `commands` and `commandsGeneral`.

Reply:
Thanks for your your feedback.
The allCommand list is now passed into KeyboardShortcutsModal as a prop.